C/manual/ch_Finance-Quote.xml

```diff
@@ -290,18 +311,38 @@ SYNOPSIS
 
     <sect2 id="fq-print-quotes">
       <title>Displaying Quotes in a Terminal Window</title>
-      <para>To display a quote for one or more stocks or the exchange rate for one or more currencies you can use <userinput>gnucash-cli --quotes dump</userinput> as follows. It offers two output forms for non-currency securities and one for currency exchange rates.
-      <itemizedlist>
-        <listitem><para>Currencies use the source <userinput>currency</userinput> and require at least two ISO-4217 currency codes; the exchange rates are denominated in the first code. For example:
+
+      <para>To display a quote for one or more stocks or the exchange rate for
+        one or more currencies you can use <userinput>gnucash-cli --quotes dump</userinput>
+        as follows. It offers two output forms for non-currency securities and
+        one for currency exchange rates.
+        <variablelist>
+        <varlistentry><term>Currencies</term>
+        
+        <listitem>
+        <para>Currencies use the source <userinput>currency</userinput> and
+          require at least two ISO-4217 currency codes; the exchange rates are
+          denominated in the first code. For example:
       <informalexample><?dbfo pgwide="1"?>
       <screen language="console">
         $ gnucash-cli --quotes dump currency USD GBP EUR
         1 GBP = 1.11917349089527 USD
 
         1 EUR = 0.9717 USD
-      </screen></informalexample></para></listitem>
-      <listitem><para>Stocks</para><itemizedlist>
-      <listitem><para>A short form displaying only the fields that GnuCash uses along with comments indicating whether the fields are optional or required; you can use this to determine if GnuCash will be able to use the quote to update your book's price database.
+      </screen></informalexample>
+      </para>
+      </listitem>
+      </varlistentry>
+
+      <varlistentry>
+      <term>Stocks</term>
+
+      <listitem>
+      <itemizedlist>
+      <listitem><para>A short form displaying only the fields that &app; uses
```

Comment:
@CWehli Please check the indentation. Currently it is hard to read.

Reply:
@fellen yes, after you reviewed #318